### ISSUE TITLE
Allow require alias

### ISSUE THREAD

**Issue by danielo515** [dylang/npm-check#248]
Hello,

Thanks for this module, works fine most of the time.
However, it will be very useful to be able to define require alias. For example,there are some places where I use proxyquire, and this module does not seem to understand this kind of requires.

Thanks and regards

**Comment by LinusU** [comment]
Could you elaborate exactly on how you are using proxyquire. I'm using it a lot for testing and haven't had any problem with it so far...

**Comment by danielo515** [comment]
Sure,

Here is an example of module that is not recogniced:

```js
const ProxyQuire = require('proxyquire').noCallThru();
const CaseNotifications = ProxyQuire('case-notifications', { './moduleLoader': () => ({ 'db-saver': DbSaver }), '@global': true });
```

**Comment by LinusU** [comment]
Aha, hmm 🤔 this is probably quite complicated since it needs to understand that the return value of a call to `proxyquire.noCallThru()` is in itself a `require`-like function...

If someone wants to take a stab at this I'd be happy to accept a PR

**Comment by danielo515** [comment]
Actually the module that is being reported as unused is 
case-notifications

**Comment by LinusU** [comment]
Yes, exactly, but for us to detect that we need to know that that `ProxyQuire` (in this example) is an require-like function, and that the first parameter sent to that (`'case-notificatios'`) should be considered a used dependency...

**Comment by danielo515** [comment]
That's why my proposal is about being able to define require aliases when you cann npm-check, to let it know which kind of functions are "require-like"
